src/Symfony/Bundle/WebProfilerBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -18,6 +18,8 @@
 class ConfigurationTest extends TestCase
 {
     /**
+     * @group legacy
```

Comment:
please don't mark the whole test as legacy, as that would schedule it for removal for 5.0. Instead, split the dataprovider in 2, one using only non-deprecated features, and one using `intercept_redirects` (with a test marked as legacy).